### DISCUSSION TITLE
LLC 2.8.0 and Polygon Theme - Error 500

### DISCUSSION THREAD

**Discussion by root9191** [JulianPrieber/littlelink-custom#159] [Issues]
Hello,
so i just updated again and tried out the new Polygon Theme, but i just get an Error 500:
![Bildschirmfoto 2022-09-11 um 14 53 40](https://user-images.githubusercontent.com/47309435/189528692-c61eaa07-4b32-4153-b54c-61b406a4e2af.png)

if i change back to Stargazer 1.1 the Icons of FB, Insta and so on are not correctly:
![Bildschirmfoto 2022-09-11 um 14 56 46](https://user-images.githubusercontent.com/47309435/189528750-43e9ac10-7aa5-4d4b-875a-034ac8299be4.png)

Can you fix this?

**Comment by JulianPrieber** [comment]
> so i just updated again and tried out the new Polygon Theme, but i just get an Error 500:

Does this error appear when updating or adding the theme?

**Reply by root9191** [reply]
The Update was successfull - The Error only appears, when i set the Polygon Theme.

**Reply by JulianPrieber** [reply]
I tested the polygon theme and ur right.

The error is pretty easy to explain, to fix it you just have to remove the -1.0 in from the folder name in your themes folder.

**Reply by JulianPrieber** [reply]
This one should work:
[polygon.zip](https://github.com/JulianPrieber/littlelink-custom/files/9542615/polygon.zip)

**Reply by root9191** [reply]
Thank you, everything solved 🤗😅
The Polygon Theme looks really cool 👌😎

**Comment by JulianPrieber** [comment]
I worked out a fix for the missing icons. I will release a hotfix shortly.

One sec.

**Reply by JulianPrieber** [reply]
https://github.com/JulianPrieber/littlelink-custom/releases/tag/v2.8.1
